src/angular/styles/_typography.scss

```diff
@@ -1539,6 +1550,8 @@ textarea {
 
   &:is(:disabled, .sbb-disabled) {
     :where(html:not(.sbb-lean)) & {
+      background-color: transparent;
```

Comment:
These two lines change the look of disabled buttons in lean mode. 
They should be removed.